src/utilities/getBmi.js

```diff
@@ -0,0 +1,12 @@
+export function getBmi(weight, height) {
```

Comment:
What are the units of the `weight` and `height` here? Centimeters, grams? Change the name of your variables to reflect that.